docs/client-specifications/error-handling.md

```diff
@@ -223,4 +223,20 @@ if (result.status === CacheSetStatus.SUCCESS) {
 }
 ```
 
+# Exceptions
 
+There are a few cases when it is appropriate to raise an exception.
+
+In general, errors inside the client or with service or about details of using Momento must be error responses. Mistakes on the part of the programmer client may be exceptions. Looking at it another way, we want to avoid crashing the whole app because of a Momento error, we only want to raise exceptions for things which should be caught during development.
```

Comment:
I'm not familiar with "programmer client." Is that a mistype?

Reply:
Yes, it's supposed to be "Mistakes on the part of the programmer"